Composer/packages/types/src/telemetry.ts

```diff
@@ -107,11 +108,29 @@ type QnaEvents = {
   AlternateQnAPhraseAdded: undefined;
 };
 
+type ResourcesItem = {
```

Comment:
We can't import ResourcesItem from where it is normally defined? Might be good to have TelemetryResourcesItem as the name if only used to recording telemetry.

Reply:
It is defined inside the extension, so outside the boundaries of the types package..